### DISCUSSION TITLE
toml.yaml file not found in deploy/fly

### DISCUSSION THREAD

**Discussion by sreejith-geodis** [supabase/supabase#26236] [General]
I have tried to install supavisor using fly but in the documentation it is mentioned that there should be a toml.yaml inside deploy/fly folder.
I downloaded it from URL https://github.com/supabase/supavisor but didn't find the toml.yaml.
Can anyone please help me to resolve this ?

**Comment by silentworks** [comment]
The reference there is to the `fly.toml` file not a `toml.yaml` file. Its a typo in the documentation. You can find the file here https://github.com/supabase/supavisor/tree/main/deploy/fly,

Edit: I've created a PR to fix the typo https://github.com/supabase/supavisor/pull/339

**Reply by sreejith-geodis** [reply]
Thank you very much for your update
